src/Symfony/Bridge/Doctrine/composer.json

```diff
@@ -40,7 +40,7 @@
         "symfony/security-core": "^5.0",
         "symfony/expression-language": "^4.4|^5.0",
         "symfony/uid": "^5.1",
-        "symfony/validator": "^5.0.2",
+        "symfony/validator": "^5.2",
```

Comment:
Is it required? Can't we skip tests when dependency is lower?

Reply:
I'm relying on the new constructor signature of the `Constraint` class and would need to duplicate some boilerplate code to support the old one. It's doable, but not worth it, imho.

Reply:
oh, thank, I mised the `parent::__construct` part. thanks